pom.xml

```diff
@@ -194,7 +204,7 @@
         <carbon.p2.plugin.version>1.5.3</carbon.p2.plugin.version>
         <carbon.identity.oauth.package.import.version.range>[6.0.0, 7.0.0)</carbon.identity.oauth.package.import.version.range>
         <carbon.identity.oauth.version>6.4.111</carbon.identity.oauth.version>
-        <carbon.identity.framework.version>5.25.215</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.25.356</carbon.identity.framework.version>
```

Comment:
this should be updated when the framework is released with your changes, right?

Reply:
Yes, this was updated for local testing. will updated once released